ui-frontend/packages/catalog-ui-search/src/main/webapp/extension-points/extension-points.tsx

```diff
@@ -53,6 +55,7 @@ const ExtensionPoints: ExtensionPointsType = {
   tableExport,
   multiSelectActions,
   userSettings,
+  customFilterInput: () => undefined,
```

Comment:
Could also define this default as the identity function to prevent the need for the check here: https://github.com/codice/ddf-ui/pull/148/files#diff-f588976ba141ff494f031e9686ccc8ebR66-R68